.changeset/fluffy-peas-care.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-backend': patch
```

Comment:
Need to add `@backstage/plugin-techdocs` too, eh?

Reply:
Correct :+1: